oep-template.rst

```diff
@@ -0,0 +1,118 @@
+=====================
+OEP-XXX: OEP Template
+=====================
+
```

Comment:
ok, reading the entire page before making any more comments...
(also, I realize that this is merged, just using this PR to make notes before we meet)